CommunityProject/Assets/_Game/Scripts/Runtime/UI/ViewBuildNumber.cs

```diff
@@ -0,0 +1,35 @@
+using BoundfoxStudios.CommunityProject.Build.BuildManifest;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using System;
+
+namespace BoundfoxStudios.CommunityProject
```

Comment:
```suggestion
namespace BoundfoxStudios.CommunityProject.UI
```

Siehe https://github.com/BoundfoxStudios/community-project/blob/develop/docs/contributing/technical/coding-conventions.md#programmierung "Nutze Namespaces"